ethereum/core/src/main/java/tech/pegasys/pantheon/ethereum/core/TransactionReceipt.java

```diff
@@ -69,32 +85,40 @@ private TransactionReceipt(
    * @param status the status code for the transaction (1 for success and 0 for failure)
    * @param cumulativeGasUsed the total amount of gas consumed in the block after this transaction
    * @param logs the logs generated within the transaction
+   * @param revertReason the revert reason
```

Comment:
@param revertReason whether to store the revert reason for failed transactions

Reply:
As above, the java doc was misleading have updated to the same as the java doc for the other constructor